python/tests/helpers/test_io.py

```diff
@@ -0,0 +1,24 @@
+"""
+Unit tests for the aac.util module.
+"""
+
+from os import getcwd
+from unittest import TestCase
+from tempfile import TemporaryDirectory
+from tests.helpers.io import new_working_dir
+
+
+class TestArchUtil(TestCase):
+    """
+    Unit test class for aac.util module.
+    """
+
+    def test_new_working_dir(self):
+        """
+        Unit test for the util.new_working_dir method.
+        """
```

Comment:
Are we going to start using docstrings in tests, now?

```suggestion
```

Reply:
It's vestigial. I don't want to.